### PR TITLE
Added some testing pipelines

### PR DESCRIPTION
**Pull request by KeManen**

### CODE REVIEW

**Comment by KeManen** [comment]
@tuomasmustakallio Could you fix the Node.js test

File: .github/workflows/backend-ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```
Comment (EddieTheCubeHead) [inline .github/workflows/backend-ci.yml:1]: Should update comment to remove mention of linting, as we do not currently lint in this workflow.

File: .github/workflows/frontend-ci.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    branches: [ "master" ]
+
+
+jobs:
+  build-frontend:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x]
```
Comment (EddieTheCubeHead) [inline .github/workflows/frontend-ci.yml:19]: @tuomasmustakallio @OttoAhlfors which node versions do we want to test?

**Review by EddieTheCubeHead — CHANGES_REQUESTED** [review]
Update comment in python CI and ensure we test correct node versions and this is good to go in my opinion.

File: .github/workflows/frontend-ci.yml (hunk 2 above)
Comment (tuomasmustakallio) [reply .github/workflows/frontend-ci.yml:19]: I'm using 20.11.0 but I think 18 works also

Comment (tuomasmustakallio) [reply .github/workflows/frontend-ci.yml:19]: Actually Node 20 LTS would probably be best on the short research(reddit thread) that I did. It has slightly better performance

**Review by EddieTheCubeHead — APPROVED** [review]

**Review by tuomasmustakallio — APPROVED** [review]

**Pull request merged**